api/src/main/java/org/openmrs/Provider.java

```diff
@@ -92,6 +96,42 @@ public String getIdentifier() {
 		return identifier;
 	}
 	
+	/**
+	 * Sets the role concept
+	 * 
+	 * @param role the role to set 
+	 */
```

Comment:
Can you add the @ since annotation to each of these new methods?

Reply:
Done